bin/gen-crons.py

```diff
@@ -7,6 +7,7 @@
 
 HEADER = '!!AUTO-GENERATED!! Edit {template}.tmpl instead.'
 TEMPLATE_DIR = os.path.abspath(os.path.join(os.path.dirname(__file__), '..', 'etc', 'cron.d'))
+LOG_DIR = '/var/log/bedrock'
```

Comment:
Not a bad plan. I created the folder manually myself just to make sure I was setting a real, usable, and sane default here. I'll add creation.